### ISSUE TITLE
Improve docs to explain what the value is when loading beams

### ISSUE THREAD

**Issue by nagordon** [sympy/sympy#15981]
In Example 5, the ramp load should be divided by length

https://docs.sympy.org/latest/modules/physics/continuum_mechanics/beam_problems.html#beam

```
# current example
b.apply_load(1, 3, 1, end=6)

# should be 
b.apply_load(1/5, 3, 1, end=6)
```

Another example can be found in the attached pdf
[Beam Ramp function incorrect in Continuum Mechanics Module.pdf](https://github.com/sympy/sympy/files/2862496/Beam.Ramp.function.incorrect.in.Continuum.Mechanics.Module.pdf)

**Comment by nileshpatra** [comment]
can I take up this issue?

**Comment by moorepants** [comment]
Here is a plot of the load for example 5:
![selection_436](https://user-images.githubusercontent.com/276007/52797096-fc86dd00-3029-11e9-9bf5-277ceb096aaf.png)

I see a slope on the ramp of 1kN/m that has a max of 3kN at the end of the beam.

Can you indicate more precisely why that might be incorrect? I don't see why at the moment.

**Comment by ishanaj** [comment]
I think there is no need to divide force by length as it is already in the form of N/m i.e. 1KN/m.
The `apply_load()` applies the load in the form of singularity function as:
```
>>> apply_load(1, 3, 1, end=6)    ----------> 1*<x-3>**1   ; where `<>` represents singularity function
```
which is similar to: 
`Piecewise((0, x < 3), (x - 3, x > 3))`

This would apply the load appropriately as needed. I don't think there is a need to divide it by length.
Although, if the resultant point load acting on the centroid of ramp load is considered as `F` then there is a need to divide it by length of the ramp.

**Comment by moorepants** [comment]
I also think it is correct.

**Comment by nagordon** [comment]
https://docs.sympy.org/latest/modules/physics/continuum_mechanics/beam_problems.html#beam
Example 5

```
  ---> x                             .
  |                                . |
  v y                    12 KN   . | |
                           |   . | | |
                           V . | | | |
\\\\|   4 KN/m             . | | | | |
\\\\|___________         . 1 KN/m  | |
\\\\|| | | | | |       . V V V V V V V
\\\\|V V V V V V     |---------------|
\\\\|________________________________
\\\\|________________________________|
\\\\|          :          :          :
\\\\|----------|-----|----|----------|
        2.0 m     1m   1m      2.0 m
>>> from sympy.physics.continuum_mechanics.beam import Beam
>>> from sympy import symbols
>>> E,I,M,V = symbols('E I M V')
>>> b = Beam(6, E, I)
>>> b.apply_load(V, 0, -1)
>>> b.apply_load(M, 0, -2)
>>> b.apply_load(4, 0, 0, end=2)
>>> b.apply_load(12, 4, -1)
>>> b.apply_load(1, 3, 1, end=6)
>>> b.solve_for_reaction_loads(V, M)
>>> b.reaction_loads
{M: 157/2, V: -49/2}
```

Do you agree the vertical reaction force should be -( 4kN/m * 2m + 12kN + 1kN/m * 3m / 2 )  = -21.5 kN

Do you agree the bending moment reaction should be 8kN * 1m + 12kN * 4m + 1.5kN * 5m = 63.5 kN * m


The example says 
```
>>> b.reaction_loads
{M: 157/2, V: -49/2} # = {M: 78.5, V: -24.5}
```
which I think is incorrect
    
To get the correct vertical reaction force and moment, you must divide the ramp magnitude by the length, eg 
```
b.apply_load(4, 0, 0, end=2)    # uniform load
b.apply_load(12, 4, -1)         # point load
b.apply_load(1/3, 3, 1 , end=6) # ramp load
```

Once solved, you get the correct result confirm with the hand calculation shown above
```
b.reaction_loads = {R1: -21.5000000000000, R2: 63.5000000000000}
```

**Comment by moorepants** [comment]
I think the issue is whether 1 kN/m listed for the ramp is the slope of the ramp or maximum load height of the ramp. It is defined as the slope of the ramp and thus the equivalent force of the ramp will be 4.5 kN @ 5 m.

**Comment by nagordon** [comment]
Ahh, which is what you get when you divide by the length. So how does that work for higher order distributed load curves like parabolic where the slope is not constant? 

Perhaps an update to the apply_load() docstring to clarify "The magnitude of an applied load or for orders greater than 0, the slope"

```
Parameters
==========
value : Sympifyable
    The magnitude of an applied load.
start : Sympifyable
    The starting point of the applied load. For point moments and
    point forces this is the location of application.
order : Integer
    The order of the applied load.
    - For moments, order= -2
    - For point loads, order=-1
    - For constant distributed load, order=0
    - For ramp loads, order=1
    - For parabolic ramp loads, order=2
    - ... so on.
end : Sympifyable, optional
    An optional argument that can be used if the load has an end point
    within the length of the beam.
```

**Comment by moorepants** [comment]
The value that is supplied is always the constant that is multiple by the respective polynomial, slope in the case of the linear case.

Yes, the docstrings can likely be improved. The design of the module is built around the singularity function method. If users aren't familiar with that, I can see what this is confusing.

**Comment by moorepants** [comment]
I'm going to change the title of this issue to reflect doc improvements.

**Comment by nagordon** [comment]
agreed, thanks for the discussion

**Comment by Abhi58** [comment]
Hello, I want to work on this issue. So do we need to improve the doc for apply_load as per [this comment](https://github.com/sympy/sympy/issues/15981#issuecomment-486815391) ?

**Comment by oscarbenjamin** [comment]
Yes please.

Take a look at the code yourself and try it out to make sure you understand what the parameters are and then see if you can make a docstring that explains it better.

**Comment by Abhi58** [comment]
Thanks @oscarbenjamin for letting me know. I opened the PR for the same #17192 please have a look.

**Comment by Psycho-Pirate** [comment]
I think this can be closed now.

**Comment by faze-geek** [comment]
I guess this issue could be closed ref #18980 .
